webapp/src/components/Form.tsx

```diff
@@ -147,6 +184,14 @@ const Form: React.FC = (props: FlexProps) => {
             election.census = census
             break
           }
+          case 'nft':
+          case 'erc20':
+            const tcensus = await axios.post(`${appUrl}/census/${data.censusType}`, {
```

Comment:
```suggestion
            const tcensus = await axios.post(`${appUrl}/census/airstack/${data.censusType}`, {
```

Shouldnt the path be `/census/airstack/erc20` or `/census/airstack/nft` ?

Reply:
And that's what it is 😕 In this part of the flow it can only be erc20 or nft, so the final urls are either:

/census/airstack/erc20
/census/airstack/nft